src/components/map-page-components/map-view/MapView.tsx

```diff
@@ -275,7 +278,9 @@ const MapView: React.FC = () => {
   return (
     <>
       <div id="pac-container">
-        <StyledSearch id="pac-input" placeholder="Address" />
+        {!(windowType === WindowTypes.Mobile) && (
```

Comment:
```suggestion
        {windowType !== WindowTypes.Mobile && (
```
This? Or, `windowType === WindowTypes.Desktop`?